ansible/vars/upp-d-delivery-eu.yaml

```diff
@@ -0,0 +1,24 @@
+debug_key_name: upp-k8s-provisioning-debug
+
+# Controller Instance Properties
+controller_instance_type: c4.large
+controller_count: 2
+
+# ETCD Instance Properties
+etcd_instance_type: t2.medium
+etcd_count: 3
+
+# Worker Instance Properties
+worker_pools: 
+  - {id: 1, instance_type: m4.xlarge, count: 3, role: worker, subnets: 3}
+  - {id: 2, instance_type: t2.medium, count: 3, role: mongo-worker, subnets: 3}
+  - {id: 3, instance_type: m4.large, count: 1, role: kafka-worker, subnets: 1}
```

Comment:
The AZ of the subnet that this is using, must be the same with the AZ where the kafka EBS volume is provisioned.

Reply:
In the delivery and the publishing yaml files, the EBS volumes for Kafka are always pinned to AZ a. Coz kafka instances are going to be spun up in AZ a